bin/API_Migration_scripts_readme/README.md

```diff
@@ -0,0 +1,24 @@
+
+# README for python scripts for API Migration 
+
+This .ipynb contains two python scripts:
+
+1. A webscrapper that scrapes the name of the package and the class for a given Android API Differences link. 
+
+2. Python script that uses the aforementioned data to cross-check and find matching java classes in a given directory.
+
+
+## Installation
+```bash
+  pip install beautifulsoup4
+  pip install requests
+```
```

Comment:
Understood. Will do it ma'am

Reply:
`conda` specifies packages in `environment.yml` file. you can also use `virtualenv` which has a `requirements.txt` instead since none of these packages appear to use `cpython`

Reply:
As you can see from the examples, please put in specific versions to avoid bitrotting the code.